app/android/app/src/main/java/com/proofofpassportapp/utils/QrCodeDetectorProcessor.kt

```
@@ -121,27 +121,95 @@ class QrCodeDetectorProcessor {
+                return result
+            } catch (e: Exception) {
+                println("Detection with hints failed with ${binarizer.javaClass.simpleName}: ${e.message}")
+            }
         }
+
+        println("All QR code detection attempts failed for bitmap ${bitmap.width}x${bitmap.height}")
         return null
     }
 
     fun stop() {
     }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)